test/functional/services/renderable.ts

```diff
@@ -26,10 +26,17 @@ export class RenderableService extends FtrService {
    */
   public async waitForRender(count: number = 1): Promise<void> {
     this.log.debug(`Renderable.waitForRender for ${count} elements`);
+    // exit if there are no objects expected
+    if (count === 0) {
+      return;
+    }
```

Comment:
If dashboard is empty, there is no need to wait for any items to be rendered.
Without this check we get stuck for 10 seconds at `const pendingElements = await this.find.allByCssSelector(RENDER_COMPLETE_PENDING_SELECTOR);` because there no elements in DOM.